api.raml

```diff
@@ -22,24 +22,17 @@ baseUri: http://api.circular.works
                       "recurrence": 2
                     },
                     "relationships": {
-                      "current_turn": {
-                        "data": {
-                          "type": "turns",
-                          "id": "3",
-                          "relationships": {
-                            "groups": {
-                              "data": [
-                                {
-                                  "type": "groups",
-                                  "id": "4",
-                                  "attributes": {
-                                    "name": "Pepe y Lola"
-                                  }
-                                }
-                              ]
-                            }
+                      "turns": {
+                        "data": [
+                          {
+                            "type": "turns",
```

Comment:
+1 turn order (`position`). Turns already have that attribute.

My only concern is if that attribute can be added in this method or ember has to load each turn separately